vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/credentialverifier/verifier/LdpVerifier.kt

```diff
@@ -29,53 +32,58 @@ import java.security.KeyFactory
 import java.security.KeyManagementException
 import java.security.NoSuchAlgorithmException
 import java.security.PublicKey
-import java.security.Signature
+import java.security.Security
 import java.security.cert.CertificateException
-import java.security.spec.MGF1ParameterSpec
-import java.security.spec.PSSParameterSpec
 import java.security.spec.X509EncodedKeySpec
 import java.util.Objects
 
-class LdpVcCredentialVerifier : CredentialVerifier {
-    private val tag: String = CredentialsVerifier::class.java.name
+class LdpVerifier {
 
-    private val util: Util = Util()
+    private val tag: String = CredentialVerifier::class.java.name
+    private var provider: BouncyCastleProvider = BouncyCastleProvider()
+
+    init {
+        Security.addProvider(provider);
+    }
```

Comment:
Can we move init after defining all the fields for more readability

Reply:
Done